index.html

```diff
@@ -1,305 +1,383 @@
+          </div>
 
-      <button type="button" class="get-my-resume">Get my resume</button>
-      <br /><br />
-      <div>
-        <div class="languages-div">
-          <h2 class="Languages">Languages</h2>
-          <div class="arrow-language">></div>
+          <div class="div-JS">
+            <ul class="icons-languages-div">
+              <li>
+                <a href="#">
+                  <img class="JS languages-icons" src="Ellipse 3.svg" alt="" />
+                </a>
+              </li>
+            </ul>
+            <span class="span-JS">CSS</span>
+          </div>
         </div>
         <br />
-        <div class="div-JS">
-          <ul class="icons-languages-div">
-            <li>
-              <a href="#">
-                <img class="JS languages-icons" src="Ellipse 1.svg" alt="" />
-              </a>
-            </li>
-          </ul>
-          <span class="span-JS">JavaScript</span>
-        </div>
-
-        <div class="div-JS">
-          <ul class="icons-languages-div">
-            <li>
-              <a href="#">
-                <img class="JS languages-icons" src="Ellipse 2.svg" alt="" />
-              </a>
-            </li>
-          </ul>
-          <span class="span-JS">HTML</span>
+        <div class="Frameworks-div">
+          <h2 class="Frameworks">Frameworks</h2>
+          <div class="Arrow-frameworks">></div>
         </div>
-
-        <div class="div-JS">
-          <ul class="icons-languages-div">
-            <li>
-              <a href="#">
-                <img class="JS languages-icons" src="Ellipse 3.svg" alt="" />
-              </a>
-            </li>
-          </ul>
-          <span class="span-JS">CSS</span>
+        <div class="Skills-div">
+          <h2 class="Skills">Skills</h2>
+          <div class="Arrow-skills">></div>
         </div>
+      </section>
 
-      </div>
-      <br />
-      <div class="Frameworks-div">
-        <h2 class="Frameworks">Frameworks</h2>
-        <div class="Arrow-frameworks">></div>
-      </div>
-      <div class="Skills-div">
-        <h2 class="Skills">Skills</h2>
-        <div class="Arrow-skills">></div>
-      </div>
-    </section>
-  </main>
-</body>
+      
+      <!-- /////////footer///////////// -->
 
-</html>
\ No newline at end of file
+      <footer class="Contact_form_container">
+        <div class="div-imges">
+          <img class="img-1" src="top-1.png" alt="" />
+          <img class="img-2" src="top2.png" alt="" />
+          <img class="img-3" src="top-3.png" alt="" />
+        </div>
+        <div class="Contact_form">
+          <div class="form_heading">
+            <h2>Contact me</h2>
+          </div>
+          <p>
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+          <form class="form" action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ ] kindly consider login in to formspree and creating your own form and integrating it into your HTML to avoid this 👇error which is showing that your form is not found 👇 
![sub](https://user-images.githubusercontent.com/60146030/220421514-e54de851-11d5-4b50-b33e-b993844fff5f.PNG)